Providers/nxOMSAutomationWorker/automationworker/worker/linuxutil.py

```diff
@@ -227,6 +227,28 @@ def get_lsb_release():
 
 
 @posix_only
+def get_oms_agent_id():
+    """Gets the oms agent id.
+
+    Returns:
+        string, the agent id
+        None,   if no agent id can be fouud
+    """
+    omsadmin_filepath = "/etc/opt/microsoft/omsagent/conf/omsadmin.conf"
```

Comment:
The resource is aware of multi-homing, it chooses to do noting for the secondary workplaces.  Support for true multi-homing will arrive later.